### DISCUSSION TITLE
wiki can't render some specific latex, and it'll affect the normal work of the program

### DISCUSSION THREAD

**Discussion by TaiiHu** [requarks/wiki#5216] [Error / Bug Report]
i copy this formula from *Tupper's self-referential* wikipedia,

`{1 \over 2}<\left\lfloor {\mathrm  {mod}}\left(\left\lfloor {y \over 17}\right\rfloor 2^{{-17\lfloor x\rfloor -{\mathrm  {mod}}(\lfloor y\rfloor ,17)}},2\right)\right\rfloor `

When render is done, it'll lead to a completely blank page.

I can't do anything with the website. In order to modify this article, I had to reopen the site and modify it directly from the admin page.

**So do not test it at you index page.**

After test, i think When a letter is in two curly brackets like `{{x}}` triggered this bug.
![image](https://user-images.githubusercontent.com/27011243/165012440-0f629bcd-cb44-45ca-bd74-5d2beba1d303.png)

**Comment by TaiiHu** [comment]
version information here
![image](https://user-images.githubusercontent.com/27011243/165013247-46c50eec-b671-4e9c-995e-76e12d6f3faa.png)

**Comment by a2k-hanlon** [comment]
I can corroborate that this is a bug. I'm running Wiki.js v2.5.285 with Node v14.18.3 and Postgres v9.2.24. I'm using the default Katex for LaTeX parsing.

To reproduce:
1. Create a new markdown page
2. Enter just the following content:

```tex
${{x}}$
```

3. Save and close the page edit. The page will render completely blank in a web browser.

This is valid LaTeX and should not cause problems, but it does. Double curly braces occur frequently in LaTeX content.

**However, I have found a workaround**: separate any pairs of identical curly braces with a space. For the above example,

```tex
${ {x} }$
```

renders successfully.

For the expression in the original question, this works:

```tex
${1 \over 2}<\left\lfloor {\mathrm {mod} }\left(\left\lfloor {y \over 17}\right\rfloor 2^{ {-17\lfloor x\rfloor -{\mathrm {mod} }(\lfloor y\rfloor ,17)} },2\right)\right\rfloor$
```
<img width="268" alt="Screen Shot 2022-07-17 at 1 28 12 PM" src="https://user-images.githubusercontent.com/55331709/179423734-b7f407c1-fa6f-46a2-a523-3c4d7d5a4b85.png">

**Reply by abhinavsood** [reply]
And while that works, just flipping the sides of the expression makes it not work:

```latex
$\left\lfloor {\mathrm {mod} }\left(\left\lfloor {y \over 17}\right\rfloor 2^{ {-17\lfloor x\rfloor -{\mathrm {mod} }(\lfloor y\rfloor ,17)} },2\right)\right\rfloor > {1 \over 2}$
```

**Comment by RingsGit** [comment]
same problem, batch splitting all '{{' and '}}'
